rfc/20250211-s3-locking-with-conditional-writes.md

```diff
@@ -0,0 +1,119 @@
+# S3 Backend: Locking by using the recently released feature of conditional writes
+
+Issue: https://github.com/opentofu/opentofu/issues/599
+
+Considering the request from the ticket above, and the newly AWS released S3 feature, we can now have state locking without relying on another instance of DynamoDB.
+
+The main reasons for such a change could be summarized as follows:
+* Less resources to maintain.
+* Removing costs that now are totally useless.
```

Comment:
* Potentially reducing costs by eliminating usage of dynamo db.

Reply:
Thanks. Fixed 711eaf70265ee42a67c97c178b4cb21326460354